### DISCUSSION TITLE
"MonadCentered" layout

### DISCUSSION THREAD

**Discussion by hamensman** [qtile/qtile#2563] [Ideas] [answered]
I think a layout like XMonad's or dwm's "CenteredMaster" would be a really cool addition to Qtile. It's the only thing missing from my ideal workflow.

Here is a demonstration (see 5:05): https://www.youtube.com/watch?v=xnREqY-oyzM

**Comment by m-col** [comment]
PRs welcome :)

**Comment by mbenford** [comment, marked as answer]
I've been working on something similar, based on the `Columns` layout. More information [here](https://github.com/qtile/qtile/discussions/2261).
